src/predict/jobs/train/models.py

```diff
@@ -77,15 +88,18 @@ def get_events(self):
 class Devices(BaseModel):
     def __init__(self):
         super().__init__()
-    
+
     def get_devices():
-  
+
         try:
-            api_url = f'{configuration.AIRQO_API_BASE_URL}devices?tenant={configuration.TENANT}&active=yes'
-            results = requests.get(api_url,verify=False)
+            api_url = f"{configuration.AIRQO_API_BASE_URL}devices?tenant={configuration.TENANT}&active=yes&network={configuration.TENANT}"
+            authorisation_token = f"'TOK:<{configuration.AIRQO_API_AUTH_TOKEN}>'"
+            results = requests.get(
+                api_url, verify=False, headers={"Authorization": authorisation_token}
```

Comment:
I hope this token has the JWT string at the beginning? Thanks @sserurich  and @123MwanjeMike

Reply:
Hi @sserurich 
We can now authenticate using access tokens as query parameters without using the JWT bearer tokens. Closing the PR...